src/open_dread_rando/files/templates/custom_init.lua

```diff
@@ -90,6 +90,14 @@ local original_Init_CreateNewGameData = Init.CreateNewGameData
 function Init.CreateNewGameData(difficulty)
     original_Init_CreateNewGameData(difficulty)
 
+    -- Mark the intro cutscenes as already played so that the intro room doesn't try to load the cutscene data
+    Blackboard.SetProp("PlayedCutscenes", "CutScenePlayed[cutscenes/0001gameintro_arrivalatrium/0001gameintro_arrivalatrium.bmscu]", "i", 1)
```

Comment:
just semantics, but could you put these cutscene IDs in a table and do this in a loop?